<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentinfo</class>
 <widget class="QDialog" name="studentinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcom back, Students!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stu_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_weekday">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_booking">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Booking for the appointment:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_booking">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Booking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bookingstatus">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NOT YET!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>608</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Booking Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>310</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CancelBooking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Time">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>Weekday</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;-Welcome To The Booking System-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BeginTime">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_EndTime">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>40</y>
     <width>29</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_time">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>160</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_endtime">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>160</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_resetPW">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ChangPW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <zorder>splitter_2</zorder>
  <zorder>label</zorder>
  <zorder>label_name</zorder>
  <zorder>label_stu_status</zorder>
  <zorder>tableView</zorder>
  <zorder>comboBox_weekday</zorder>
  <zorder>label_booking</zorder>
  <zorder>pushButton_booking</zorder>
  <zorder>label_bookingstatus</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_Time</zorder>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>label_BeginTime</zorder>
  <zorder>label_EndTime</zorder>
  <zorder>comboBox_time</zorder>
  <zorder>label_endtime</zorder>
  <zorder>pushButton_resetPW</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>studentinfo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>801</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
